### PR TITLE
add ServerTokens & ServerSignature configuration

### PR DESCRIPTION
**Pull request by yfouillet**
Hi,

I added the options "ServerTokens" and "ServerSignature" with the highest security settings. This allow a security by design with the apache best practice.

### CODE REVIEW

**Comment by goldpanic** [comment]
I wish this commit would be merged.

File: tasks/configure-Debian.yml [hunk 1]
```diff
@@ -8,6 +8,22 @@
   with_items: "{{ apache_ports_configuration_items }}"
   notify: restart apache
 
+- name: Configure granularity of apache version displayed.
+  lineinfile:
+    dest: "{{ apache_server_root }}/{{ apache_daemon }}.conf"
```
Comment (hspaans) [inline tasks/configure-Debian.yml:13]: On Debian 10 this setting is in `{{ apache_server_root }}/conf-available/security.conf` by default.

File: tasks/configure-Debian.yml [hunk 2]
```diff
@@ -8,6 +8,22 @@
...
+    regexp: "^ServerTokens "
+    line: "ServerTokens {{ apache_config_version_visibility }}"
+    state: present
+  notify: restart apache
+
+- name: Configure granularity of apache signature displayed.
+  lineinfile:
+    dest: "{{ apache_server_root }}/{{ apache_daemon }}.conf"
```
Comment (hspaans) [inline tasks/configure-Debian.yml:21]: On Debian 10 this setting is in `{{ apache_server_root }}/conf-available/security.conf` by default.

**Comment by steinbrueckri** [comment]
Hi guys,
can someone reopen the PR? I would need that too :D

**Comment by nicovs** [comment]
i think you could also use the following settings:

```yaml
apache_global_vhost_settings: |
  DirectoryIndex index.php index.html
  ServerSignature Off
  ServerTokens Prod
```

the site-enabled folder is the last folder with configs to be included.

**Pull request closed without merging**